cg/meta/archive/archive.py

```diff
@@ -0,0 +1,42 @@
+import logging
+from collections import namedtuple
+from typing import Dict, List
+
+
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.meta.archive.ddn_dataflow import DDNDataFlowApi
+from cg.store import Store
+
+LOG = logging.getLogger(__name__)
+
+PathAndSample = namedtuple("PathAndSample", "path sample_internal_id")
+
+
+class ArchiveAPI:
+    """Class handling the archiving of sample SPRING files."""
```

Comment:
Maybe this class doc string can be extended to explain what archiving means? I have no clue 😅